Readme.md

```diff
@@ -34,9 +41,17 @@ Bug reports, feature requests, comments, questions, code contributions and love
 
 This project is developed by [Lombiq Technologies](https://lombiq.com/). Commercial-grade support is available through Lombiq.
 
+### Default .NET version
+
+For .NET workflows, the default .NET SDK version we should provide is a concrete patch version of the latest .NET version, the most recent one at the time of updating .NET support (e.g., to `8.0.301` when .NET 8 is the latest). We need to pin the .NET SDK to a specific version like this to avoid unexpected build changes that patch version updates bring (what happens if the version is specified as e.g. `8.0.301`). See [this issue](https://github.com/dotnet/roslyn/issues/73639) for more context.
```

Comment:
```suggestion
For .NET workflows, the default .NET SDK version we should provide is a concrete patch version of the latest .NET version, the most recent one at the time of updating .NET support (e.g., to `8.0.301` when .NET 8 is the latest). We need to pin the .NET SDK to a specific version like this to avoid unexpected build changes that patch version updates bring (which happens if the version is specified as e.g. `8.0.x`). See [this issue](https://github.com/dotnet/roslyn/issues/73639) for more context.
```